particles/Tutorial/Kotlin/Demo/TTTGame.kt

```diff
@@ -0,0 +1,173 @@
+package arcs.tutorials
+
+import arcs.Collection
+import arcs.Handle
+import arcs.Particle
+import arcs.Singleton
+import arcs.TTTGame_Events
+import arcs.TTTGame_GameState
+import arcs.TTTGame_PlayerOne
+import arcs.TTTGame_PlayerOneMove
+import arcs.TTTGame_PlayerTwo
+import arcs.TTTGame_PlayerTwoMove
+import kotlin.native.internal.ExportForCppRuntime
+
+class TTTGame : Particle() {
+    private val gameState = Singleton { TTTGame_GameState() }
+    private val playerOne = Singleton { TTTGame_PlayerOne() }
+    private val playerOneMove = Singleton { TTTGame_PlayerOneMove() }
+    private val playerTwo = Singleton { TTTGame_PlayerTwo() }
+    private val playerTwoMove = Singleton { TTTGame_PlayerTwoMove() }
+    private val events = Collection { TTTGame_Events() }
+
+    private val winningSequences = arrayOf(
+        arrayOf(0, 1, 2),
+        arrayOf(3, 4, 5),
+        arrayOf(6, 7, 8),
+        arrayOf(0, 3, 6),
+        arrayOf(1, 4, 7),
+        arrayOf(2, 5, 8),
+        arrayOf(0, 4, 8),
+        arrayOf(2, 4, 6)
+    )
+
+    private val defaultGame = TTTGame_GameState(
+        board = ",,,,,,,,",
+        currentPlayer = (0..1).random().toDouble(),
+        gameOver = false
+    )
+
+    init {
+        registerHandle("gameState", gameState)
+        registerHandle("playerOne", playerOne)
+        registerHandle("playerOneMove", playerOneMove)
+        registerHandle("playerTwo", playerTwo)
+        registerHandle("playerTwoMove", playerTwoMove)
+        registerHandle("events", events)
+    }
+
+    override fun onHandleSync(handle: Handle, allSynced: Boolean) {
+        if (gameState.get()?.board == null) {
+            gameState.set(defaultGame)
+        }
+        if (handle.name.equals("playerOne") && playerOne.get()?.id != 0.0) {
```

Comment:
Prefer `==` to `.equals` here and elsewhere in this PR where string equality is evaluated (Kotlin is different from Java here, in that `==` is overridden to call `.equals` for you).